debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/J9BCUtil.java

```diff
@@ -261,7 +261,7 @@ private static void dumpModifiers(PrintStream out, long modifiers, int modifierS
 				if (ONLY_SPEC_MODIFIERS == modScope) {
 					modifiers &= J9CfrClassFile.CFR_FIELD_ACCESS_MASK;
 				} else {
-					modifiers &= J9CfrClassFile.CFR_FIELD_ACCESS_MASK | J9FieldFlags.J9FieldFlagConstant;
+					modifiers &= J9CfrClassFile.CFR_FIELD_ACCESS_MASK | J9FieldFlags.J9FieldFlagConstant | J9FieldFlags.J9FieldFlagIsNullRestricted;
```

Comment:
Please update the comment for this method so it mentions `J9FieldFlagConstant` and `J9FieldFlagIsNullRestricted` in the set of valid modifiers for fields.

Reply:
updated.